e2e/playwright.config.ts

```diff
@@ -35,16 +36,27 @@ export default defineConfig({
     {
       name: 'chromium',
       use: { ...devices['Desktop Chrome'] },
+      testIgnore: '**',
```

Comment:
Is this temporary? Whats the long term plan here?

Reply:
After re-investigating this, I don't think this works for multiple browsers. I'll come up with a better solution.

Reply:
Project dependencies did not work how I thought. I've changed the way this works. Now, projects only select browser, and frontend is selected via directory. Each top-level directory will have its own fixtures file that sets the base URL via the environment (this will allow us to modify the URL for dev/test/local as needed).